lib/Lift/JumpTargetManager.h

```diff
@@ -596,6 +614,26 @@ class JumpTargetManager {
   MetaAddressSet AVIPCWhiteList;
   const TupleTree<model::Binary> &Model;
   const RawBinaryView &BinaryView;
+
+private:
```

Comment:
Reorder the following functions according to the order in which they are invoked in `harvestWithAVI`